components/automate-chef-io/content/docs/configuration.md

```diff
@@ -379,6 +379,17 @@ Uncomment and change settings as needed, and then run `chef-automate config patc
 # key = "-----BEGIN RSA PRIVATE KEY-----\n<your load balancer private key>\n-----END RSA PRIVATE KEY-----\n"
 ```
 
+#### Buffer Size
```

Comment:
Would it be helpful to the user to provide context around when they should consider changing this?

Reply:
Let's get it documented & we can follow up later